src/toastify.js

```diff
@@ -54,6 +54,8 @@
       this.options.className = options.className || ""; // additional class names for the toast
       this.options.stopOnFocus = options.stopOnFocus === undefined? true: options.stopOnFocus; // stop timeout on focus
       this.options.onClick = options.onClick; // Callback after click
+      this.options.containerOffset = options.containerOffset || options.containerOffset === 0 ? options.containerOffset : 15; // Starting top/bottom spacing to wrap container
+      this.options.offset = options.offset || options.offset === 0 ? options.offset : 15; // Spacing between toasts
```

Comment:
Do we need this check here? `options.offset === 0`. I think it's better to check whether the type is a number.

Reply:
Yeah, that makes sense. I improved the validation of these attributes.